assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/outputs.tf

```diff
@@ -62,3 +62,7 @@ output "kubelet_values" {
 output "calico_values" {
   value = module.bootkube.calico_values
 }
+
+output "lokomotive_values" {
```

Comment:
Sorry for going back and forth, but IMO the most reasonable name for the chart would be `lokomotive-operator`, instead of just `lokomotive`.